### ISSUE TITLE
x/website: /blog/feed.atom missing an author field

### ISSUE THREAD

**Issue by gudvinr** [golang/go#68869]
### What is the URL of the page with the issue?
 
https://go.dev/blog/feed.atom
 
 
### What is your user agent?
 

 
Mozilla/5.0 (X11; Linux x86_64; rv:129.0) Gecko/20100101 Firefox/129.0

 
 
### Screenshot
 

 
n/a
 
 
### What did you do?
 

 
1. Download an Atom feed
 
### What did you expect to see?

`<author></author>` for an entry that has same value as an author of corresponding article.

Example:
- https://go.dev/blog/go1.23 has an author "Dmitri Shuralyov, on behalf of the Go team"
- atom entry with `<author><name>Dmitri Shuralyov</name></author>`
 
 
### What did you see instead?

Atom entry for https://go.dev/blog/go1.23 contains `<author><name/></author>`

[atom specs](https://validator.w3.org/feed/docs/atom.html) has the following:
-  A feed may have multiple author elements. _A feed must contain at least one author element unless all of the entry elements contain at least one author element._
- `<author>` and `<contributor>` describe a person, corporation, or similar entity. _It has one required element, name_, and two optional elements: uri, email.
- `<name>` conveys a human-readable name for the person.

So, in conclusion, feed.atom doesn't follow the spec.  
While `<entry>` elements do have `<author>` elements and _technically_ `<name>` is filled, it is filled by nothing which is not "a human-readable name for the person"

**Comment by cagedmantis** [comment]
cc [@golang/tools-team](https://github.com/orgs/golang/teams/tools-team)
